pkg/component/generic/collection/v0/.compogen/bottom.mdx

```diff
@@ -0,0 +1,64 @@
+
+
+## Example Recipes
+
+In this example, we are combining two arrays of strings using the `union` and `concat` components.
+```yaml
+variable:
```

Comment:
How about documenting and providing examples with a comment here? E.g.

```yaml
# This pipeline combines 2 arrays of strings
# Examples:
# ["foo", "bar"], ["zot", "bat"] -> ["foo", "bar", "zot", "bat"], []

Reply:
I added these 2 in corresponding components.
```
# This component concatenates two arrays of strings.
# Examples:
# ["foo", "bar"], ["zot", "bat"] -> ["foo", "bar", "zot", "bat"]
```


```
# This component combines two arrays of strings in a union, ensuring that no duplicates are included.
# Examples:
# ["foo", "bar"], ["foo", "bat"] -> ["foo", "bar", "bat"]
```

Reply:
I meant documented the input/output of the **pipelines**, not the individual components. But it's fine.